compose/services.yml

```diff
@@ -73,23 +73,24 @@ services:
       BALENA_ROOT_CA: ${OPENBALENA_ROOT_CA}
       BALENA_TOKEN_AUTH_ISSUER: api.${OPENBALENA_HOST_NAME}
       BALENA_TOKEN_AUTH_REALM: https://api.${OPENBALENA_HOST_NAME}/auth/v1/token
-      COMMON_REGION:
+      COMMON_REGION: ${OPENBALENA_S3_COMMON_REGION}
```

Comment:
This is called `COMMON_REGION` because it's reused between components in balenaCloud. The variable itself need not include `COMMON`, or `S3`, so my suggestion is to change it to `OPENBALENA_S3_REGION` or `OPENBALENA_COMMON_REGION`. I'm slightly in favour of the latter as it's closer to how it's used in balenaCloud.

Reply:
@dfunckt agreed - I didnt rename it as I wasn't sure of any balenaCloud impact. I would prefer it referenced S3 as it makes it obvious as to which component will be using it.

Reply:
@richbayliss sorry, I think my comment was confusing. What I meant was something like the following:

```
COMMON_REGION: ${OPENBALENA_COMMON_REGION}
```

`COMMON_REGION` is best left alone as like you say it's needed by the balenaCloud API.